api/instance.go

```diff
@@ -145,6 +145,7 @@ type MigrationHistory struct {
 	Statement           string             `jsonapi:"attr,statement"`
 	Schema              string             `jsonapi:"attr,schema"`
 	SchemaPrev          string             `jsonapi:"attr,schemaPrev"`
+	LastSchema          string             `jsonapi:"attr,lastSchema"`
```

Comment:
LastRecordedSchema

Also please comment the difference between SchemaPrev and this field here since they are inherently confusing